frontend/src/css/modules/edit-challenge.scss

```diff
@@ -2,8 +2,12 @@
 	padding: 20px;
 }
 
-.width-500 {
-	width: 500px;
+.width-550 {
+	width: 550px;
```

Comment:
I don't think making it fixed will help on larger screens. You might want to set it by `%` so that it can be proportionate according to the screen resolution.